notebooks/how_to/run_unit_metrics.ipynb

```diff
@@ -0,0 +1,1369 @@
+      ],
+      "text/plain": [
+       "XGBClassifier(base_score=None, booster=None, callbacks=None,\n",
+       "              colsample_bylevel=None, colsample_bynode=None,\n",
+       "              colsample_bytree=None, early_stopping_rounds=10,\n",
+       "              enable_categorical=False, eval_metric=['error', 'logloss', 'auc'],\n",
+       "              feature_types=None, gamma=None, gpu_id=None, grow_policy=None,\n",
+       "              importance_type=None, interaction_constraints=None,\n",
+       "              learning_rate=None, max_bin=None, max_cat_threshold=None,\n",
+       "              max_cat_to_onehot=None, max_delta_step=None, max_depth=None,\n",
+       "              max_leaves=None, min_child_weight=None, missing=nan,\n",
+       "              monotone_constraints=None, n_estimators=100, n_jobs=None,\n",
+       "              num_parallel_tree=None, predictor=None, random_state=None, ...)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_df, validation_df, test_df = demo_dataset.preprocess(raw_df)\n",
+    "\n",
+    "x_train = train_df.drop(demo_dataset.target_column, axis=1)\n",
+    "y_train = train_df[demo_dataset.target_column]\n",
+    "x_val = validation_df.drop(demo_dataset.target_column, axis=1)\n",
+    "y_val = validation_df[demo_dataset.target_column]\n",
+    "\n",
+    "model = xgb.XGBClassifier(early_stopping_rounds=10)\n",
+    "model.set_params(\n",
+    "    eval_metric=[\"error\", \"logloss\", \"auc\"],\n",
+    ")\n",
+    "model.fit(\n",
+    "    x_train,\n",
+    "    y_train,\n",
+    "    eval_set=[(x_val, y_val)],\n",
+    "    verbose=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "pandas.core.series.Series"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(test_df['Exited'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['CreditScore',\n",
+       " 'Gender',\n",
+       " 'Age',\n",
+       " 'Tenure',\n",
+       " 'Balance',\n",
+       " 'NumOfProducts',\n",
+       " 'HasCrCard',\n",
+       " 'IsActiveMember',\n",
+       " 'EstimatedSalary',\n",
+       " 'Geography_France',\n",
+       " 'Geography_Germany',\n",
+       " 'Geography_Spain']"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "feature_columns = [col for col in test_df.columns if col != demo_dataset.target_column]\n",
+    "feature_columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute Predictions"
```

Comment:
Can you add some short text description after this title?

Reply:
Done.